packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -621,6 +621,14 @@ const conf = (module.exports = convict({
     env: 'PAGE_TEMPLATE_SUBDIRECTORY',
     format: ['src', 'dist'],
   },
+  nimbus: {
+    host: {
+      default: 'http://localhost:8001',
+      doc: 'Base URI for cirrus (Nimbus experimentation endpoint).',
+      env: 'NIMBUS_CIRRUS_HOST',
```

Comment:
We still need to ask SRE to add this env to stage/prod before landing I suspect.

Reply:
A PR has been filed, and it's on their radar.